terraform/aws/projects/nasa-veda.tfvars

```diff
@@ -18,11 +18,71 @@ hub_cloud_permissions = {
   "staging" : {
     requestor_pays: true,
     bucket_admin_access: ["scratch-staging"],
-    extra_iam_policy: ""
+    extra_iam_policy: <<-EOT
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:PutObject",
+                    "s3:GetObject",
+                    "s3:ListBucketMultipartUploads",
+                    "s3:AbortMultipartUpload",
+                    "s3:ListBucketVersions",
+                    "s3:CreateBucket",
+                    "s3:ListBucket",
+                    "s3:DeleteObject",
+                    "s3:GetBucketLocation",
+                    "s3:ListMultipartUploadParts"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::veda-data-store-staging",
+                    "arn:aws:s3:::veda-data-store-staging/*"
+                ]
+            },
+            {
+                "Effect": "Allow",
+                "Action": "s3:ListAllMyBuckets",
+                "Resource": "*"
+            }
+        ]
+    }
+  EOT
   },
   "prod" : {
     requestor_pays: true,
     bucket_admin_access: ["scratch"],
-    extra_iam_policy: ""
+    extra_iam_policy: <<-EOT
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Effect": "Allow",
+                "Action": [
+                    "s3:PutObject",
+                    "s3:GetObject",
+                    "s3:ListBucketMultipartUploads",
+                    "s3:AbortMultipartUpload",
+                    "s3:ListBucketVersions",
+                    "s3:CreateBucket",
+                    "s3:ListBucket",
+                    "s3:DeleteObject",
+                    "s3:GetBucketLocation",
+                    "s3:ListMultipartUploadParts"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::veda-data-store-staging",
```

Comment:
Wanted to confirm - this is the same buckat as for staging and prod. Is that intentional? If so, this looks fine to me.

Reply:
Hi Yuvi ✨  Yes, intentional